run_file.py

```diff
@@ -0,0 +1,70 @@
+import prewl, json
```

Comment:
This file shouldn't be included in the repo...just for your personal testing. It's all in the README already.

If you really want short snippets to demonstrate different functionality, you can create an `examples/` directory.

Reply:
Just for testing purposes I should move this file then? Should the examples/ directory be in PREWL?

Reply:
Yes, this file should appear in the root directory. If you wanted to do the `examples/` directory, then you could place something like this in there. Small test files to show different functionality (one for gpt2, one for the callback function to check the output, etc).

Reply:
I created an examples directory and put run_file in it. I will change its name to show that it is testing gpt2